### PR TITLE
Add model index files + generator

### PR DESCRIPTION
**Pull request by bwasty**
Adds `model-index.json` files and a Python script to generate them. See #77 for background.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Really cool, thanks @bwasty!

@emackey did you want to look at this?

@lexaknyazev is this easy to connect to CI?

**Comment by bwasty** [comment]
@emackey I've just rebased and re-generated the indexes since I want to use them for a new viewer I'm working on.

**Comment by emackey** [comment]
Thanks @bwasty.

File: generate-index.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        os.chdir(model)
+        model_contents = os.listdir(".")
+        gltf_variant_dirs = [d for d in model_contents if d.startswith("glTF")]
+        variants = {}
+        for variant_dir in gltf_variant_dirs:
+            model_file = [f for f in os.listdir(variant_dir)
+                          if f.endswith(".glb") or f.endswith(".gltf")][0]
+            variants[variant_dir] = model_file
+        if "screenshot" not in model_contents:
+            print "WARNING: no screenshot found for {}".format("model")
```
Comment (emackey) [inline generate-index.py]: So I just merged this, and then I found a problem.  On line 21 here, `model` should not be in quotes.  Also, Python 3.x requires the `print` arguments to be in parentheses, as it changed from a statement to a function.  Can you open a new PR for that?  Thanks!

File: generate-index.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        index.append({
+            "name": model,
+            "variants": variants,
+            "screenshot": "screenshot/" + os.listdir("screenshot")[0]
```
Comment (emackey) [inline generate-index.py]: Also, without an `else` for the above conditional, the missing screenshot folder will fail the script here.

File: generate-index.py (hunk 1 above)
Comment (bwasty) [reply generate-index.py]: Ok, will do.

Comment (emackey) [reply generate-index.py]: Addressed in #168.

File: generate-index.py (hunk 2 above)
Comment (emackey) [reply generate-index.py]: Addressed in #168.

**Pull request merged**